packages/client/components/InsightsDomainNudge.tsx

```diff
@@ -132,12 +139,13 @@ const InsightsDomainNudge = (props: Props) => {
               <BoldText>{domainId}</BoldText>
               {` is over the limit of `}
               <BoldText>{`${Threshold.MAX_STARTER_TIER_TEAMS} free teams`}</BoldText>
-              {toBeLockedOrg?.scheduledLockAt && (
+              {scheduledLockAt && !isLocked && (
                 <>
                   {`. Your free access will end in `}
-                  <BoldText>{`${relativeDate(toBeLockedOrg?.scheduledLockAt)}.`}</BoldText>
+                  <BoldText>{`${relativeDate(scheduledLockAt)}.`}</BoldText>
                 </>
               )}
+              {isLocked && `. Please upgrade to continue using Parabol`}
```

Comment:
+1 could add a full stop at the end of this sentence: `Please upgrade to continue using Parabol.`